src/common/raft/integration/raft_knowhere_index.cuh

```diff
@@ -0,0 +1,620 @@
+/**
```

Comment:
This file provides the templated implementation of the underlying logic for dispatching from Knowhere to RAFT. It is not included in any Knowhere header, however, allowing us to completely separate RAFT from Knowhere at this layer.